### PR TITLE
Add specific by hostname metrics that don't have other labels

### PR DESCRIPTION
**Pull request by glennslaven**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-# install tooling
-RUN go get -v \
-  github.com/kisielk/errcheck \
-  golang.org/x/lint/golint
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  ca-certificates \
+  git \
+  unzip
+
+RUN update-ca-certificates
```
Comment (jstangroome) [inline Dockerfile:10]: nit: if we're using recent Golang base images, it should already contain sufficiently recent CA certs

File: p8s.go [hunk 2]
```diff
@@ -107,6 +120,12 @@ func InitMetrics(additionalLabels ...string) *prometheus.Registry {
 		requestLabels = append(requestLabels, geoHash)
 	}
 
+	if idx := slices.Index(sanitizedP8sLabels, hostnameLabel); idx > -1 {
```
Comment (jstangroome) [inline p8s.go:123]: nit:
>A little copying is better than a little dependency.

https://go-proverbs.github.io/

```go
func Index[E comparable](s []E, v E) int {
	for i, vs := range s {
		if v == vs {
			return i
		}
	}
	return -1
}
```
[src](https://cs.opensource.google/go/x/exp/+/master:slices/slices.go;l=104-113)

**Review by jstangroome — APPROVED** [review]

File: p8s.go (hunk 2 above)
Comment (glennslaven) [reply p8s.go:123]: So then we have various permutations of that function that subtly differ across our 50 different apps. No, I pretty fundamentally disagree with that proverb.  left-pad notwithstanding I've never yet been convinced the costs of repeated copying is cheaper than the costs of dependency.

File: Dockerfile (hunk 1 above)
Comment (glennslaven) [reply Dockerfile:10]: Turns out we don't need any of those dependencies.

**Pull request merged**